pallets/connectors/src/message.rs

```diff
@@ -273,27 +282,71 @@ mod tests {
 			};
 			let encoded = msg.encode();
 
-			let input = "0400000000000000010000000000000000000000000000000101010101010101010101010101010101010101010101010101010101010101016400000000000000";
+			let input = "0400000000000000010000000000000000000000000000000101010101010101010101010101010101010101010101010101010101010101010000000000000064";
 			let expected = <[u8; 65]>::from_hex(input).expect("Decoding failed");
 			assert_eq!(encoded, expected);
 		}
 
 		#[test]
-		#[ignore = "wip"]
-		fn transfer() {
+		fn update_member_that_failed() {
+			let msg = Message::<Domain, PoolId, TrancheId, Balance, Rate>::UpdateMember {
+				pool_id: 2,
+				tranche_id: <[u8; 16]>::from_hex("811acd5b3f17c06841c7e41e9e04cb1b").expect(""),
+				address: <[u8; 32]>::from_hex(
+					"1231231231231231231231231231231231231231231231231231231231231231",
+				)
+				.expect(""),
+				valid_until: 1706260138,
+			};
+			let encoded = msg.encode();
+
+			let input = "040000000000000002811acd5b3f17c06841c7e41e9e04cb1b12312312312312312312312312312312312312312312312312312312312312310000000065b376aa";
+			let expected = <[u8; 65]>::from_hex(input).expect("Decoding failed");
+			assert_eq!(hex::encode(encoded), hex::encode(expected));
```

Comment:
Why do we the expected hex into bytes and back into hex? Below we just compare `hex::encode(encoded)` with `input`.

Reply:
Thanks for the catch. This was legacy code, so most tests were inconsistent in that regard. All standardized now 👍